pages/tokens/[type].tsx

```diff
@@ -43,9 +43,7 @@ const TokenList = () => {
     { key: 'address', label: 'address' },
     { key: type === 'bridge' ? 'circulatingSupply' : 'totalSupply' },
     { key: 'holderCount' },
-    // TODO: in split UAN feature, add origin and bridge for bridge token
-    // { key: 'origin' },
-    // { key: 'bridge' },
+    ...(type === 'bridge' ? [{ key: 'origin' }, { key: 'bridge' }] : []),
```

Comment:
`Native token` could also be a `bridged` token(namely it's a bridged native token), it could be bridged from other layer2 with celer bridge https://v1.gwscan.com/tokens/native

Reply:
> > @Keith-CY For txs list component, I saw it's using `SimplePagination` , but the pagination used in `/txs` page is different, I am not sure whether it's possible to adapt shared TxList component to use that kind of pagination
> 
> Could reuse the simple pagination, actually it almost ready in refine-tx-page branch [godwoken-explorer-ui-git-refine-tx-detail-nervosnet.vercel.app/txs](https://godwoken-explorer-ui-git-refine-tx-detail-nervosnet.vercel.app/txs)

The PR has been merged @qiweiii